automation-scripts/minibroker-charts/extractor.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Script to extract working versions of minibroker database charts from helm stable repo's index.yaml.
+# Make sure the YAML parsing lib is installed: `pip install ruamel.yaml`.
+# 
+# Usage:
+# 1. Update the versions.yaml with the minibroker DB versions known to be working with current CAP release.
+# 2. Run `python extractor.py`
+
+import os
+from ruamel.yaml import YAML
+
+os.system("curl -s https://kubernetes-charts.storage.googleapis.com/index.yaml > helm_stable_index.yaml")
```

Comment:
- Has python a http package which could be used over spawning a sub process ?
- Use a temp file for the result.
- Move into `main`